langchain/experimental/plan_and_execute/executors/agent_executor.py

```diff
@@ -12,15 +12,29 @@
 
 {agent_scratchpad}"""
 
+TASK_PREFIX = """{objective}
+
+"""
+
 
 def load_agent_executor(
-    llm: BaseLanguageModel, tools: List[BaseTool], verbose: bool = False
+    llm: BaseLanguageModel,
+    tools: List[BaseTool],
+    verbose: bool = False,
+    include_task_in_prompt: bool = False,
 ) -> ChainExecutor:
+    input_variables = ["previous_steps", "current_step", "agent_scratchpad"]
+    TEMPLATE = HUMAN_MESSAGE_TEMPLATE
```

Comment:
nit: since local variable, should be lowercase (`template`)

Reply:
Done :)